src/lib/es5.d.ts

```diff
@@ -2,6 +2,8 @@
 /// ECMAScript APIs
 /////////////////////////////
 
+// tslint:disable no-var-keyword

```

Comment:
not sure i agree that we should ship our library with these comments. i would rather we exclude them from the lint.